include/psen_scan_v2/scanner_controller.h

```diff
@@ -127,7 +128,7 @@ ScannerControllerT<TCSM, TUCI>::ScannerControllerT(const ScannerConfiguration& s
 template <typename TCSM, typename TUCI>
 void ScannerControllerT<TCSM, TUCI>::handleUdpData(const MaxSizeRawData& data, const std::size_t& num_bytes)
 {
-  MonitoringFrameMsg frame{ MonitoringFrameMsg::deserialize(data, num_bytes) };
+  MonitoringFrameMsg frame{ deserialize(data, num_bytes) };
```

Comment:
I think the function name is ambiguous, because `data` could also contain a start/stop reply.